default.hbs

```diff
@@ -60,6 +66,13 @@
     <script src="{{asset "main/js/util.js"}}"></script>
     <script src="{{asset "main/js/main.js"}}"></script>
 
+    <script src="{{asset "main/js/ghostHunter.js"}}"></script>
```

Comment:
Following https://github.com/AdvisorySG/advisory-editorial/commit/ff73fcd05fccbbb4fc9284235c11f743bbf716be, please change the file location to `ghostHunter/src/ghosthunter.js`. You'll also need to include jQuery, think slim minified version from https://code.jquery.com/ should do the trick:

```html
<script
    src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha256-pasqAKBDmFT4eHoN2ndd6lN370kFiGUFyTiUHWhU7k8="
    crossorigin="anonymous"></script>
```

Reply:
Could you also include the latest version of jQuery, instead of using the one bundled with the theme? Think slim minified version from https://code.jquery.com/ should do the trick:

```html
<script
    src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha256-pasqAKBDmFT4eHoN2ndd6lN370kFiGUFyTiUHWhU7k8="
    crossorigin="anonymous"></script>
```